app/views/api/v1/followers/users.json.jbuilder

```diff
@@ -3,5 +3,5 @@ json.array! @follows do |follow|
   json.id         follow.id
   json.created_at utc_iso_timestamp(follow.created_at)
 
-  json.partial! "api/v0/shared/follows", user: follow.follower
+  json.partial! "api/v1/shared/follows", user: follow.follower
```

Comment:
I noticed this was a copy/paste error in v1 endpoints. This meant v1 jbuilder views were rendering v0 partials. I updated this and there shouldn't be any noticeable difference because they're both the same (partials) in v0 & v1. This just makes partial usage consistent.